metpy/calc/tests/test_indices.py

```diff
@@ -19,3 +19,33 @@ def test_precipitable_water():
     pw = precipitable_water(data.variables['dewpoint'][:], data.variables['pressure'][:])
     truth = (0.8899441949243486 * units('inches')).to('millimeters')
     assert_array_equal(pw, truth)
+
+
+def test_mean_wind_pressure_weighted():
+    """Test pressure-weighted mean wind function with vertical interpolation."""
+    with UseSampleData():
+        data = get_upper_air_data(datetime(2016, 5, 22, 0), 'DDC', source='wyoming')
+    mean6 = mean_wind_pressure_weighted(data.variables['u_wind'][:],
+                                        data.variables['v_wind'][:],
+                                        data.variables['pressure'][:],
+                                        data.variables['height'][:],
+                                        depth=6000 * units('meter'))
+    truth = [6.0208700094534775, 7.966031839967931] * units('m/s')
```

Comment:
Better just to put the truth values directly in the `assert_almost_equal` calls below rather than putting them into an array, only to break it apart.